python/spacewalk/satellite_tools/repo_plugins/deb_src.py

```diff
@@ -341,6 +341,7 @@ def __init__(self, url, name, insecure=False, interactive=True, yumsrc_conf=None
             self.num_packages = 0
             self.num_excluded = 0
 
+            self.urlgrabber_logspec = None
```

Comment:
Why do we need to set the parameter for `yum_src` plugin, but always keeping as `None` for `deb_src`?

Reply:
Actually I mean not to delete this part from deb plugin, but to extend it the same way as you did for rpm version. It was just incomplete as the value from the config was not used to fill the parameter.